supertokens_python/recipe/emailverification/ev_claim.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
```

Comment:
rename this file to ev_claim_validators.py

Reply:
done